src/components/Books/ProgressBar.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+const ProgressBar = () => {
+  const [percentage, setPercentage] = useState(Math.floor(Math.random() * (100 - 0 + 1) + 0));
+
+  const updatePercentage = () => {
+    if (percentage + 10 > 100) {
+      setPercentage(100);
+    } else {
+      setPercentage(percentage + 10);
+    }
+  };
+
+  return (
+    <div className="user-progress-container">
+      <div className="user-progress">
+        <div
+          className="progressbar"
+          style={{
+            width: '4.25rem',
+            height: '4.25rem',
+          }}
```

Comment:
- [OPTIONAL] As requested by the previous code reviewer, please kindly define these styles in a different file and import them in to this component, for inline styling is not a best practice.